pkg/terraform/signals.go

```diff
@@ -1,3 +1,18 @@
+// Copyright 2020 The Lokomotive Authors
```

Comment:
This is more than what the commit msg says. Please add a mention of it in the commit message or put it in a different commit.

Reply:
Yes, this should have been part of the first commit. Done.